package.json

```diff
@@ -7,11 +7,15 @@
     "eslint": "^5.16.0",
     "eslint-plugin-react": "^7.13.0",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6"
+    "react-dom": "^16.8.6",
+    "react-redux": "^7.1.0",
+    "redux": "^4.0.1",
+    "redux-thunk": "^2.3.0"
   },
   "scripts": {
     "develop": "webpack-dev-server",
-    "build": "webpack --mode production"
+    "build": "webpack --mode production",
+    "coveralls": "jest --detectOpenHandles --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
What does this script do?

Reply:
@kburudi requested that I should add it for coverage to be recorded. His branch doesn't have a `package.json`